swap_meet/decor.py

```diff
@@ -1,2 +1,14 @@
-class Decor:
-    pass
\ No newline at end of file
+### WAVE 5
+from .item import Item
+
+class Decor(Item): # Updated class to inherit from Item
+    def __init__(self, id=None, width=0, length=0, condition=0):
+        super().__init__(id, condition)  # pass id and condition to Item, move 
+                                        # the line above child's custom attribute
+        self.width = 0 if width == 0 else width
+        self.length = 0 if length == 0 else length
```

Comment:
While this will set `width` or `length` to be 0 if the variables are passed in with the default value of 0 and it will set the attributes to be the values of `width` and `length` if they aren't default values, it's a little wonky. We should invert the logic.

Recall that 0 is a falsey value so we don't need to check that something is `==` to 0.

```suggestion
        self.width = width if width else 0
        self.length = length if length else 0
```

Similar to what you wrote on line 7 in `Clothing`

Reply:
Got it! That makes sense!